Common/Sources/Extensions/HyperwalletFee.swift

```diff
@@ -95,6 +110,8 @@ public extension HyperwalletFee {
                 if let value = value {
                     feeFormat = "fee_percent_no_min_and_max_formatter".localized()
                     description = String(format: feeFormat, value)
+                } else {
+                    description = FeeTypes.noFee.feeDescription()
```

Comment:
why this one uses `FeeTypes.noFee.feeDescription()` and the in flatFeeDescription it's `else {
                description = noFeeDescription()
            }` Just keep it consistent

Reply:
Updated with `FeeTypes.noFee.feeDescription()` to have consistency